### PR TITLE
Fix exceptions to be compatible with Python 3.

### PR DESCRIPTION
**Pull request by kibertoad**
refs #1098

### CODE REVIEW

**Comment by caryoscelus** [comment]
I would suggest rewriting commit message to put issue reference into message body rather than title.

**Comment by kibertoad** [comment]
Done.

**Comment by caryoscelus** [comment]
Eh, i meant commit message, not pr.

**Comment by Mirv** [comment]
You're very speedy kibertoad!  

I am not really involved in the coding yet, but I like to read what is going on!  I saw in the `execution.py` you had used literal numbers for the arrays.  Is there a way to loop over that which would make it so any additions or removals would not need to be hand coded?

**Comment by caryoscelus** [comment]
@Mirv, it's just results of `sys.exc_info()` call. You aren't supposed to add or remove anything from it anyway.

File: renpy/execution.py [hunk 1]
```diff
@@ -458,7 +458,7 @@ def run(self, node=None):
                     except renpy.game.CONTROL_EXCEPTIONS as ce:
                         raise ce
                     except Exception as ce:
-                        raise exc_info[0], exc_info[1], exc_info[2]
+                        raise exc_info[0](exc_info[1]).with_traceback(exc_info[2])
```
Comment (jsfehler) [inline renpy/execution.py]: This only works in python3

File: .gitignore [hunk 2]
```diff
@@ -38,6 +38,7 @@ styles.txt
 /.pydevproject.bak
 /.project
 /.cproject
+/.idea
```
Comment (jsfehler) [inline .gitignore]: This doesn't really have anything to do with exceptions or python3

**Comment by tcyrus** [comment]
Shouldn't this be closed since #1596 was merged

**Comment by renpytom** [comment]
Closing.

**Pull request closed without merging**